### ISSUE TITLE
homebridge-camera-ffmpeg installation overwrites ffmpeg

### ISSUE THREAD

**Issue by normen** [KhaosT/homebridge-camera-ffmpeg#460]
**Describe The Bug:**

When installing homebridge-camera-ffmpeg on a Rapberry Pi with a self-compiled ffmpeg in /usr/local/bin it will overwrite that ffmpeg version.

**To Reproduce:**

- Self-compile and install ffmpeg with prefix /usr/local
- Install homebridge-camera-ffmpeg through config-ui-x

**Expected behavior:**

The installer should notice theres a version of ffmpeg already installed in /usr/local and refrain from overwriting it

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Config:**

```json
Show your homebridge config.json here.
Remove any sensitive information, such as your homebridge-gsh / google-smarthome token.
```

**Screenshots:**

**Environment:**

* **Node.js Version**: 
v10.15.3

* **NPM Version**: 
6.14.4

* **Homebridge Version**: 
0.4.53

* **Homebridge Camera FFmpeg Version**:
0.1.17

* **Homebridge Config UI X Plugin Version**:
4.15.1

* **Operating System**: 
Raspbian GNU/Linux 9 (stretch)

**Comment by milmber** [comment]
I believe this is by design. From 0.1.15 there is a dependency on pre-compiled ffmpeg binaries from the homebridge `ffmpeg-for-homebridge` project: https://github.com/homebridge/ffmpeg-for-homebridge

See https://github.com/KhaosT/homebridge-camera-ffmpeg/compare/v0.1.14...v0.1.15

**Comment by normen** [comment]
I believe so too but theres people that want to use their own compiled version of ffmpeg..

**Comment by Edsonytic** [comment]
Indeed I believe the newer versions have made my setup harder to use, with other programs dependent on FFmpeg being affected.
Couldn’t this instance be limited to be shown only to homebridge or this plugin? Or even better just having the option to either use this recompiled package vs whatever is already running in our setup.

**Comment by NorthernMan54** [comment]
The bundling of ffmpeg with the plugin does not replace the OS installed version.  Not sure what the issue is?

Os version is here on my RPI

/usr/bin/ffmpeg

Plugin version is here

homebridge-camera-ffmpeg/node_modules/ffmpeg-for-homebridge/ffmpeg

**Comment by normen** [comment]
You're completely right, there is no issue I'm sorry, PICNIC.

I had a self-compiled version with some esoteric audio codecs in `/usr/local` but I must have replaced it myself at some point. When I tested the new version of camera-ffmpeg one camera failed to produce an audio stream. But apparently I simply didn't stream from that camera in a long time and only noticed the issue now.

Again, sorry. Closing this.
